src/main/java/teammates/logic/api/Logic.java

```diff
@@ -301,6 +301,15 @@ public void putDocument(InstructorAttributes instructor) {
     }
 
     /**
+     * Batch creates or updates documents for the given Instructors.
+     *
+     * @see InstructorsLogic#putDocuments(List)
+     */
+    public void batchPutInstructorDocuments(List<InstructorAttributes> instructors) {
```

Comment:
hmm... I cannot use putDocuments because type erasure causes all of them take in `List<EntityAttributes>`, thats why I changed the method name. What do you guys think is the best way to proceed?